### ISSUE TITLE
Home page has 2 h1 elements

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/earthworks#1504]
<img width="686" alt="Image" src="https://github.com/user-attachments/assets/9570ec56-793f-4f01-8bae-601e5af7e8f2" />

**Comment by thatbudakguy** [comment]
This is gonna get complicated quick. The 'Earthworks' in the masthead _isn't_ an H1 despite looking like one, so we have two H1s that should be H2s, but if they were, the hierarchy would be wrong because they'd be without an H1 above them.

That holds for the homepage, but on show pages, the item's title is the H1 (which I think is a Blacklight default), so we can't have the masthead be the H1 there.

Maybe we:
- do nothing on show pages
- add a hidden h1 on the homepage so that the section titles can be H2s?

Having to conditionally render the masthead as an H1 some places but not others would suck.
